package.json

```diff
@@ -4,10 +4,11 @@
   "description": "x11 javascript hashing algorithm in pure javascript",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js",
+    "test": "mocha --timeout 10000 --full-trace --check-leaks test/test.js",
```

Comment:
@andyfreer In the future I would prefer if we split this into another PR as this isn't directly related to the .eslint changes (per the [contribution guidelines written in Dash Core](https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#pull-request-philosophy)).

Reply:
ok